src/driver.h

```diff
@@ -3,9 +3,10 @@
 
 #include "drivers/inc/stm32f4xx.h"
 #include "bus.h"
+#include <stdbool.h>
 
 CANlib_Transmit_Error_T CANlib_TransmitFrame(Frame *frame, CANlib_Bus_T bus);
-void CANlib_ReadFrame(Frame *frame, CANlib_Bus_T bus);
+bool CANlib_ReadFrame(Frame *frame, CANlib_Bus_T bus);
```

Comment:
Maybe introduce `CANlib_Receive_Error_T`, idk if we need to mess with this now tho